SofaKernel/modules/SofaBaseCollision/SofaBaseCollision_test/Sphere_test.cpp

```diff
@@ -126,8 +118,8 @@ bool TestSphere::rigidRigid1(){
 
     sofa::helper::vector<DetectionOutput> detectionOUTPUT;
 
-    //loooking for an intersection
-    if(!BaseIntTool::computeIntersection(sph1,sph2,1.0,1.0,&detectionOUTPUT))
+    //looking for an intersection (with proximities)
+    if(!m_proxIntersection->computeIntersection(sph1,sph2,&detectionOUTPUT))
```

Comment:
no check if m_proxIntersection is not null

Reply:
m_proxIntersection is created/instanciated in the Setup() function (for sure it wont be null) so I dont really see the point to test if it is null 🤔